### ISSUE TITLE
Setting fixedOverflowWidgets does not seem to work anymore

### ISSUE THREAD

**Issue by spahnke** [Microsoft/monaco-editor#1203]
**monaco-editor version:** 0.15.5
**Browser:** any
**OS:** Windows 10

Hi, 

we embed the editor in a div which has `overflow: hidden` set which I can't change easily (it's a custom element coming from our framework). In 0.14.3 you were able to set `fixedOverflowWidgets: true`  in this case to always show widgets like the context menu or the suggest widget to their full extent. This does not work anymore in 0.15.5.

**Playground example:**
Open a suggest widget or context menu at the border of the marked div.
```js
monaco.editor.create(document.getElementById("container"), {
    value: "function hello() {\n\talert('Hello world!');\n}",
    language: "javascript",
    fixedOverflowWidgets: true
});
```

```html
<div style="height: 300px; overflow: hidden; border: 1px solid red;">
    <div id="container" style="height:100%;"></div>
</div>
```

Thanks!

**Comment by alexandrudima** [comment]
@spahnke Not sure I fully understand the problem. Here is what I get (which looks kind of OK):

![image](https://user-images.githubusercontent.com/5047891/48782849-556eff80-ecdf-11e8-9a89-0984777816e2.png)

**Comment by spahnke** [comment]
Ah, that looks right, but it is broken for the context menu. I will edit the issue text.
![grafik](https://user-images.githubusercontent.com/25029871/48785475-0d52db80-ece5-11e8-9e1a-1923a0b0113f.png)

**Comment by alexandrudima** [comment]
Might be related to #1199. But @spahnke AFAIK the flag `fixedOverflowWidgets` was never influencing the context menu (in any editor version).

**Comment by spahnke** [comment]
You are right, the flag didn't affect the context menu at all. I don't where I was looking...
Is there anything else I could do about that?

I'm so sorry for the noise! 😥

**Comment by joaomoreno** [comment]
Hm... we probabaly need an option in the editor in order not to let the context menu overflow the editor space.

**Comment by spahnke** [comment]
That would be ideal.

**Comment by izikl** [comment]
@alexdima was there any progress on this issue?

**Comment by Weffe** [comment]
@alexdima Just pinging to see if this backlog item could be addressed some time in the near future.

**Comment by lidermanrony** [comment]
ping on this

**Comment by jardenliu** [comment]
pined
